python/ql/src/semmle/python/frameworks/Stdlib.qll

```diff
@@ -1542,6 +1542,45 @@ private module Stdlib {
   class Sqlite3 extends PEP249Module {
     Sqlite3() { this = sqlite3() }
   }
+
+  // ---------------------------------------------------------------------------
+  // re
+  // ---------------------------------------------------------------------------
+  private module Re {
+    /** List of re methods. */
+    private class ReMethods extends string {
+      ReMethods() { this in ["match", "fullmatch", "search", "split", "findall", "finditer"] }
+    }
+
+    /** re.ReMethod(pattern, string) */
+    private class DirectRegex extends DataFlow::Node {
```

Comment:
```suggestion
    private class DirectRegex extends RegexExecution::Range {
```

This will automatically make `DirectRegex` considered when looking for `RegexExecution`s.